### PR TITLE
[JW8-2310] Fix background being transparent when floating

### PR DESCRIPTION
**Pull request by pajong**
### Are there any Pull Requests open in other repos which need to be merged with this?
https://github.com/jwplayer/jwplayer-commercial/pull/5807

#### Addresses Issue(s):
JW8-2310

### CODE REVIEW

**Comment by pajong** [comment]
test this please

File: src/css/jwplayer/flags/floatingplayer.less [hunk 1]
```diff
@@ -2,6 +2,10 @@
     background-size: cover;
     overflow: visible;
 
+    &.jwplayer {
```
Comment (vseventer) [inline src/css/jwplayer/flags/floatingplayer.less]: This can be on `.jw-flag-floating` directly right?

File: src/css/jwplayer/states.less [hunk 2]
```diff
@@ -2,7 +2,9 @@
 
 // setup
 .jw-state-setup {
-    background-color: transparent;
+    .jw-wrapper {
+        background-color: transparent;
```
Comment (vseventer) [inline src/css/jwplayer/states.less]: Maybe nicer to use `inherit`.

**Comment by pajong** [comment]
test this please

**Pull request merged**